webpack.config.js

```diff
@@ -273,6 +274,10 @@ if ( isDevelopment ) {
 	webpackConfig.entry.build = path.join( __dirname, 'client', 'boot', 'app' );
 }
 
+webpackConfig.plugins.push(
```

Comment:
since we want this in every environment there isn't a reason to `push` it to the array.
Might as well include it in the plugins definition

Reply:
done!